### DISCUSSION TITLE
UNO Platform - Pair to Mac for iOS development requirements?

### DISCUSSION THREAD

**Discussion by ward-horsfall** [unoplatform/uno#15991] [Q&A] [answered]
Hi,

In regards to using the UNO platform to develop for iOS what is the more specific hardware requiremnts for the Pair to Mac.

Pair to Mac for iOS development
https://learn.microsoft.com/en-us/dotnet/maui/ios/pair-to-mac?view=net-maui-8.0

In the Microsoft article it does not specify the minimun requirements for MacOS or the hardware requirents needed (RAM / CPU / Disk).

eg. Do I macOS 14 Somona or can it work with something earlier?

Thanks,

Ward

**Comment by jeromelaban** [comment, marked as answer]
The requirements for working on a mac are tied to the version of .NET iOS, which is also tied to the requirements for Xcode. The latest version of .NET 8 requires Xcode 15.2, which itself requires macOS Sonoma. For the hardware requirements, most recent supported machines for Sonoma are good enough to build iOS apps.
